robot/board.py

```diff
@@ -155,17 +155,17 @@ def receive(self, should_retry=True):
 
         return json.loads(line.decode('utf-8'))
 
-    def send_and_receive(self, message, should_retry=True):
+    def _send_and_receive(self, message, should_retry=True):
         """
         Send a message to robotd and wait for a response.
         """
-        self.send(message, should_retry)
-        return self.receive(should_retry)
+        self._send(message, should_retry)
+        return self._receive(should_retry)
 
-    def close(self):
```

Comment:
I'd argue against changing this one - `close` (not `_close`) is the conventional name of the method that releases a resource. I think that's a valid reason for it not to have an underscore prefix.

Reply:
Alternatively, `Robot.close` should be changed as well.